samples/zigbee/light_switch/README.rst

```diff
@@ -51,12 +58,11 @@ Button 2:
     If the brightness level is decreased to the minimum, the effect of turning on the light bulb might not be noticeable.
 
 Button 3:
-    When pressed while resetting the board, enables the Sleepy End Device behavior.
+    When pressed while resetting the board, enables the :ref:`zigbee_ug_sed`.
```

Comment:
@sebastiandraus , lines 60-61 will be removed in #2329 , because SED is mentioned in the user guide, while it shouldn't be mentioned yet. When merging this PR, please make sure these lines are added.